fastly/s3.go

```diff
@@ -8,35 +8,40 @@ import (
+	S3ServerSideEncryptionKMS S3ServerSideEncryption = "aws:kms"
 )
 
 // S3 represents a S3 response from the Fastly API.
 type S3 struct {
 	ServiceID string `mapstructure:"service_id"`
 	Version   int    `mapstructure:"version"`
 
-	Name              string       `mapstructure:"name"`
-	BucketName        string       `mapstructure:"bucket_name"`
-	Domain            string       `mapstructure:"domain"`
-	AccessKey         string       `mapstructure:"access_key"`
-	SecretKey         string       `mapstructure:"secret_key"`
-	Path              string       `mapstructure:"path"`
-	Period            uint         `mapstructure:"period"`
-	GzipLevel         uint         `mapstructure:"gzip_level"`
-	Format            string       `mapstructure:"format"`
-	FormatVersion     uint         `mapstructure:"format_version"`
-	ResponseCondition string       `mapstructure:"response_condition"`
-	MessageType       string       `mapstructure:"message_type"`
-	TimestampFormat   string       `mapstructure:"timestamp_format"`
-	Placement         string       `mapstructure:"placement"`
-	Redundancy        S3Redundancy `mapstructure:"redundancy"`
-	CreatedAt         *time.Time   `mapstructure:"created_at"`
-	UpdatedAt         *time.Time   `mapstructure:"updated_at"`
-	DeletedAt         *time.Time   `mapstructure:"deleted_at"`
+	Name                         string                 `mapstructure:"name"`
+	BucketName                   string                 `mapstructure:"bucket_name"`
+	Domain                       string                 `mapstructure:"domain"`
+	AccessKey                    string                 `mapstructure:"access_key"`
+	SecretKey                    string                 `mapstructure:"secret_key"`
+	Path                         string                 `mapstructure:"path"`
+	Period                       uint                   `mapstructure:"period"`
+	GzipLevel                    uint                   `mapstructure:"gzip_level"`
+	Format                       string                 `mapstructure:"format"`
+	FormatVersion                uint                   `mapstructure:"format_version"`
+	ResponseCondition            string                 `mapstructure:"response_condition"`
+	MessageType                  string                 `mapstructure:"message_type"`
+	TimestampFormat              string                 `mapstructure:"timestamp_format"`
+	Placement                    string                 `mapstructure:"placement"`
+	Redundancy                   S3Redundancy           `mapstructure:"redundancy"`
+	CreatedAt                    *time.Time             `mapstructure:"created_at"`
+	UpdatedAt                    *time.Time             `mapstructure:"updated_at"`
+	DeletedAt                    *time.Time             `mapstructure:"deleted_at"`
+	ServerSideEncryptionKMSKeyID string                 `mapstructure:"server_side_encryption_kms_key_id"`
+	ServerSideEncryption         S3ServerSideEncryption `mapstructure:"server_side_encryption"`
```

Comment:
Minor nit: please place these above the `CreatedAt` etc metadata fields to follow convention with the rest of the project.